kernel/prepare/pkg.yaml

```diff
@@ -5,15 +5,15 @@ dependencies:
   - image: '{{ .TOOLS_IMAGE }}'
 steps:
   - sources:
-      - url: https://cdn.kernel.org/pub/linux/kernel/v{{ regexReplaceAll ".\\d+\\.\\d+$" .linux_version "${1}" }}.x/linux-{{ .linux_version }}.tar.xz
+      - url: https://git.kernel.org/torvalds/t/linux-6.12-rc7.tar.gz
         destination: linux.tar.xz
```

Comment:
Would make sense to also update this :)

Reply:
this is a draft PR, it is not going to merge this way. 

we are not updating to 6.12 at the moment: https://github.com/siderolabs/talos/discussions/9743

Reply:
Ah sorry for lurking here; I didn't notice that FAQ. I'm trying to get a fork of 6.12 working with Talos on the Raspberry Pi (#7978) and was wondering in case I could piggy-back on this PR. For the moment the lack of ZFS or NVIDIA modules are OK for a small Pi setup.